src/program/lwaftr/run_nohw/run_nohw.lua

```diff
@@ -17,45 +17,49 @@ local function check(flag, fmt, ...)
 end
 
 local function parse_args(args)
-   local verbosity = 0
+   local opts = {
+      verbosity = false,
+   }
    local conf_file, b4_if, inet_if
-   local bench_file = 'bench.csv'
    local handlers = {
       v = function ()
-         verbosity = verbosity + 1
-      end;
+         opts.verbosity = true
+      end,
       c = function (arg)
          check(file_exists(arg), "no such file '%s'", arg)
          conf_file = arg
-      end;
+      end,
```

Comment:
They worked. Lua allows to separate table items with semicolons but it's not common. In most of the codebase, commas are used instead.

I think it was @aperezdc who wrote this code. Maybe he can explain better what was the reason for using semicolons :)